dali/operators/segmentation/random_object_bbox.cc

```diff
@@ -404,6 +457,30 @@ bool RandomObjectBBox::PickForegroundBox(SampleContext &context) {
   return ret;
 }
 
+void RandomObjectBBox::AllocateTempStorage(const TensorVector<CPUBackend> &input) {
+  int64_t max_blob_bytes = 0;
+  int64_t max_filtered_bytes = 0;
+  int N = input.ntensor();
+  for (int i = 0; i < N; i++) {
+    int64_t vol = input[i].size();
+    int label_size = vol > 0x80000000 ? 8 : 4;
```

Comment:
It'd be more readable if you use numeric_limits and sizeof here

Reply:
I think that 0x80000000 (which, by the way, is not a 32-bit constant by virtue of C++ promoting it to long to accommodate a big signed number) is more readable than:
```
static_cast<int64_t>(std::numeric_limits<int32_t>::max()) + 1
```